src/creditcard/CreditCard.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import './CreditCard.css';
+
+const CreditCard = props => {
```

Comment:
All the information is there, so it's good for me! The styling can be done later :D